endorser/tests/test_endorser_processor.py

```diff
@@ -161,13 +168,18 @@ async def test_is_valid_issuer_x_res_errors(mocker: MockerFixture):
     assert not await is_valid_issuer(did, schema_id)
 
     # Error schema res
-    mocked_async_client.get = AsyncMock(
-        side_effect=[
-            actor_res,
-            Response(400, json={}),
-        ]
+    not_schema_id = "not-the-schema-id"
+    error_response = Response(status_code=400)
+    error_response.raise_for_status = Mock(
+        side_effect=HTTPStatusError(
+            response=error_response,
+            message="Something went wrong",
+            request=not_schema_id,
+        )
     )
-    assert not await is_valid_issuer(did, schema_id)
+    mocked_async_client.get = AsyncMock(side_effect=[actor_res, error_response])
```

Comment:
`actor_res, error_response` - consistency would be better, to use _response everywhere instead of _res too